### PR TITLE
Adds APIM Analytics Overview page

### PR DESCRIPTION
**Pull request by tishan89**
## Purpose
$Subject

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: en/docs/observe/api-manager-analytics/overview-of-api-analytics.md [hunk 1]
```diff
@@ -1,221 +1,45 @@
...
-</tr>
-</table>
-
-## Customizing the Analytics Dashboards
-
-Apart from the latter mentioned default dashboards, the users can create custom dashboards based on their specific requirements as explained in **[Customizing Analytics Dashboards]({{base_path}}/observe/api-manager-analytics/customizing-analytics-dashboards)**.
+![APIM Analytics Overview Page]({{base_path}}/assets/img/observe/apim-analytics-overview-page.png)
+
+APIs are widely used in Business Integrations and are becoming a key part of driving an organization's business 
+strategy. Ensuring the proper functioning of the APIs, getting feedback on the performance and deriving business
```
Comment (samgnaniah) [inline en/docs/observe/api-manager-analytics/overview-of-api-analytics.md:5]:
```suggestion
strategy. Ensuring the proper functioning of the APIs, getting feedback on the performance, and deriving business 
```

File: en/docs/observe/api-manager-analytics/overview-of-api-analytics.md [hunk 2]
```diff
@@ -1,221 +1,45 @@
...
-## Customizing the Analytics Dashboards
-
-Apart from the latter mentioned default dashboards, the users can create custom dashboards based on their specific requirements as explained in **[Customizing Analytics Dashboards]({{base_path}}/observe/api-manager-analytics/customizing-analytics-dashboards)**.
+![APIM Analytics Overview Page]({{base_path}}/assets/img/observe/apim-analytics-overview-page.png)
+
+APIs are widely used in Business Integrations and are becoming a key part of driving an organization's business 
+strategy. Ensuring the proper functioning of the APIs, getting feedback on the performance and deriving business 
+insights from APIs are becoming equally important.
+
+With the API Manager 4.0.0 release API Analytics was completely revamped and offered as a cloud-based analytics solution
```
Comment (samgnaniah) [inline en/docs/observe/api-manager-analytics/overview-of-api-analytics.md:8]:
```suggestion
With the API Manager 4.0.0 release, API Analytics was completely revamped and offered as a cloud-based analytics solution
```

File: en/docs/observe/api-manager-analytics/overview-of-api-analytics.md [hunk 3]
```diff
@@ -1,221 +1,45 @@
...
+ to provide users with an unparallel analytics experience addressing many customer concerns we received for earlier
```
Comment (samgnaniah) [inline en/docs/observe/api-manager-analytics/overview-of-api-analytics.md:9]:
```suggestion
 to provide users with an unparalleled analytics experience addressing many customer concerns we received for earlier 
```
Shall we avoid mention of customer concerns here? We don't normally introduce that sort of viewpoint in docs.

File: en/docs/observe/api-manager-analytics/overview-of-api-analytics.md [hunk 4]
```diff
@@ -1,221 +1,45 @@
...
+strategy. Ensuring the proper functioning of the APIs, getting feedback on the performance and deriving business 
+insights from APIs are becoming equally important.
+
+With the API Manager 4.0.0 release API Analytics was completely revamped and offered as a cloud-based analytics solution
+ to provide users with an unparallel analytics experience addressing many customer concerns we received for earlier 
+ releases.
+
+## Highlights
+### Hassle Free Setting Up
+One of the major concerns customers had earlier was the effort taken up to setup the analytics environment. This
```
Comment (samgnaniah) [inline en/docs/observe/api-manager-analytics/overview-of-api-analytics.md:14]: Same comment as before. Let's remove this line.

File: en/docs/observe/api-manager-analytics/overview-of-api-analytics.md [hunk 5]
```diff
@@ -1,221 +1,45 @@
...
+## Highlights
+### Hassle Free Setting Up
+One of the major concerns customers had earlier was the effort taken up to setup the analytics environment. This 
+involves setting up a minimum of two node deployment with tuned parameters which is a considerable effort and utilizes 
+considerable amount of resources. 
+With all new API Manager 4.0.0 cloud-based analytics model all that you need to do is,
+ - [Register for API Manager Analytics]({{base_path}}/observe/api-manager-analytics/configure-analytics/register-for-analytics)
+ - [Configure Synapse Gateway]({{base_path}}/observe/api-manager-analytics/configure-analytics/configure-synapse-gateway) or [Choreo Connect Gateway]({{base_path}}/observe/api-manager-analytics/configure-analytics/configure-microgateway)
+
+These are the only two steps that is needed to setup analytics with the new solution
```
Comment (samgnaniah) [inline en/docs/observe/api-manager-analytics/overview-of-api-analytics.md:21]:
```suggestion
These are the only two steps that are needed to setup analytics with the new solution.
```

File: en/docs/observe/api-manager-analytics/overview-of-api-analytics.md [hunk 6]
```diff
@@ -1,221 +1,45 @@
...
+
+## Design
+All above highly user-friendly features are possible through the new design adopted by the analytics solution. 
+Below diagram illustrate the basic architecture of the solution.
+
+![APIM Analytics Simplified Design]({{base_path}}/assets/img/observe/apim-analytics-simplified.jpg)
+
+As you can observe Gateways will now publish analytics stat directly to analytics cloud over the internet. 
+Analytics cloud will have regional deployments to reduce publishing latencies and honor data privacy. 
+Analytics console is deployed as a separate web application which will query the processing later to visualize
```
Comment (samgnaniah) [inline en/docs/observe/api-manager-analytics/overview-of-api-analytics.md:44]:
```suggestion
Analytics console is deployed as a separate web application that will query the processing to visualize
```

File: en/docs/observe/api-manager-analytics/overview-of-api-analytics.md [hunk 7]
```diff
@@ -1,221 +1,45 @@
...
+ the cloud-based analytics now you can configure alerts using the same console you used to view analytics. 
+ Please refer this documentation to configure alerts
+
+## Design
+All above highly user-friendly features are possible through the new design adopted by the analytics solution. 
+Below diagram illustrate the basic architecture of the solution.
+
+![APIM Analytics Simplified Design]({{base_path}}/assets/img/observe/apim-analytics-simplified.jpg)
+
+As you can observe Gateways will now publish analytics stat directly to analytics cloud over the internet.
```
Comment (samgnaniah) [inline en/docs/observe/api-manager-analytics/overview-of-api-analytics.md:42]:
```suggestion
As you can observe, Gateways will now publish analytics stats directly to the analytics cloud over the internet. 
```

File: en/docs/observe/api-manager-analytics/overview-of-api-analytics.md [hunk 8]
```diff
@@ -1,221 +1,45 @@
...
+  you can achieve through analytics portal.
+
+### One Portal to View and Configure Alerts
+In the earlier versions customers had to log in to a separate console to configure alerts related to analytics. With
+ the cloud-based analytics now you can configure alerts using the same console you used to view analytics. 
+ Please refer this documentation to configure alerts
+
+## Design
+All above highly user-friendly features are possible through the new design adopted by the analytics solution. 
+Below diagram illustrate the basic architecture of the solution.
```
Comment (samgnaniah) [inline en/docs/observe/api-manager-analytics/overview-of-api-analytics.md:38]:
```suggestion
The following diagram illustrates the basic architecture of the solution.
```

File: en/docs/observe/api-manager-analytics/overview-of-api-analytics.md [hunk 9]
```diff
@@ -1,221 +1,45 @@
...
+### Modern Responsive UIs
+Earlier versions on API Manager analytics relied on home drown dashbaording solution for which We received the
+ feedback that UIs look outdated. With cloud-based analytics now we have sleek and modern responsive UIs with improved 
+ filtering and drill down capabilities. 
+You can access the all new analytics portal through here. Use this document as a reference to composition of each
+ dashboard and information that is available through them. This document will guide you through different usecases
+  you can achieve through analytics portal.
+
+### One Portal to View and Configure Alerts
+In the earlier versions customers had to log in to a separate console to configure alerts related to analytics. With
```
Comment (samgnaniah) [inline en/docs/observe/api-manager-analytics/overview-of-api-analytics.md:32]:
```suggestion
In the earlier versions, customers had to log on to a separate console to configure alerts related to analytics. With
```

File: en/docs/observe/api-manager-analytics/overview-of-api-analytics.md [hunk 10]
```diff
@@ -1,221 +1,45 @@
...
+involves setting up a minimum of two node deployment with tuned parameters which is a considerable effort and utilizes 
+considerable amount of resources. 
+With all new API Manager 4.0.0 cloud-based analytics model all that you need to do is,
+ - [Register for API Manager Analytics]({{base_path}}/observe/api-manager-analytics/configure-analytics/register-for-analytics)
+ - [Configure Synapse Gateway]({{base_path}}/observe/api-manager-analytics/configure-analytics/configure-synapse-gateway) or [Choreo Connect Gateway]({{base_path}}/observe/api-manager-analytics/configure-analytics/configure-microgateway)
+
+These are the only two steps that is needed to setup analytics with the new solution
+
+### Modern Responsive UIs
+Earlier versions on API Manager analytics relied on home drown dashbaording solution for which We received the
```
Comment (samgnaniah) [inline en/docs/observe/api-manager-analytics/overview-of-api-analytics.md:24]: Let's remove this as well.

File: en/docs/observe/api-manager-analytics/overview-of-api-analytics.md [hunk 11]
```diff
@@ -1,221 +1,45 @@
...
+ feedback that UIs look outdated. With cloud-based analytics now we have sleek and modern responsive UIs with improved 
+ filtering and drill down capabilities. 
+You can access the all new analytics portal through here. Use this document as a reference to composition of each
```
Comment (samgnaniah) [inline en/docs/observe/api-manager-analytics/overview-of-api-analytics.md:27]:
```suggestion
You can access the all new analytics portal through this solution. Use this document as a reference to the composition of each
```

File: en/docs/observe/api-manager-analytics/overview-of-api-analytics.md (hunk 4 above)
Comment (tishan89) [reply en/docs/observe/api-manager-analytics/overview-of-api-analytics.md]: Fixed with ceadce23591c5187ed8bf6e0e424b49e64832bcf

File: en/docs/observe/api-manager-analytics/overview-of-api-analytics.md (hunk 10 above)
Comment (tishan89) [reply en/docs/observe/api-manager-analytics/overview-of-api-analytics.md]: Fixed with ceadce23591c5187ed8bf6e0e424b49e64832bcf

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**